File: STSCapitalApi/Entity/Models/Task.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace DataAccess.Models
{
    public partial class Task
    {
        public int  TaskId { get; set; }
        public string TaskGuid { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime TaskDate { get; set; }
        public DateTime TaskFromTime { get; set; }
        public DateTime TaskToTime { get; set; }
        public string Location { get; set; }
        public bool IsDeleted { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}

File: STSCapitalUI/src/app/modules/feature/task/models/task.ts


export interface ITask {
    taskGuid?: string;
    title: string;
    description?: string;
    taskDate: Date;
    taskFromTime?: Date;
    taskToTime?: Date;
    location?: string;
}

export const createTaskModel = ({
    taskGuid = '',
    title = '',
    description = '',
    taskDate = new Date(),
    taskFromTime = new Date(),
    taskToTime = new Date(),
    location = ''
}: ITask = {
        title: '',
        taskDate: new Date()
    }): ITask => ({
        taskGuid,
        title,
        description,
        taskDate,
        taskFromTime,
        taskToTime,
        location
    });
